package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "new-scrubbing",
+  "version": "1.0.0",
+  "description": "Video preview or scrubbing without backend workflow",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "find ./src -maxdepth 1 -iname '*.js' -not -name 'ampPlayer.js' -exec cat {} +> src/static/js/bundle.js && cp src/ampPlayer.js src/static/js/"
```

Comment:
This is really clever! I'm having two questions:

1. Is the `find` command works on both Windows and Linux?
2. Can you add the minifier step?

Reply:
This build step will only work in bash. I can write a js script to move & minify that should work fine xplat, is it ok to do that in an additional PR to unblock Dan's build pipeline experimentation?

Reply:
Yes, agree. Let's capture this in another task.